fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1360,6 +1360,7 @@ cancelBackupStatement
 
 showBackupStatement
     : SHOW BACKUP ((FROM | IN) identifier)?
+    | SHOW ALL BACKUP
```

Comment:
What's the difference between `show all backup` and `show backup`?

Reply:
show all backup will show all backup info from all different database with backup task. But show backup just show the info of current used database.